worker/src/RTC/NackGenerator.cpp

```diff
@@ -121,9 +120,8 @@ namespace RTC
 			if (it != this->recoveredList.begin())
 				this->recoveredList.erase(this->recoveredList.begin(), it);
 
-			// Do not let a packet pass if it's newer than last seen seq and came via
-			// RTX.
-			return false;
+			// Let the packet pass if it's newer than last seen seq and came via RTX.
```

Comment:
Please add a comment indicating that some implementations use RTX for probation and such packets may arrive before the corresponding RTP packet.